fedimint-core/src/util/mod.rs

```diff
@@ -319,42 +323,42 @@ pub fn handle_version_hash_command(version_hash: &str) {
 ///   error of the closure is returned
 pub async fn retry<F, Fut, T>(
     op_name: impl Into<String>,
+    statergy: impl backon::BackoffBuilder,
```

Comment:
technically this is an API break and we'd be breaking SemVer. Likely not a huge deal, but want you to be aware @dpc.